### DISCUSSION TITLE
Multiple Servers?

### DISCUSSION THREAD

**Discussion by seigel** [Dokploy/dokploy#22] [Q&A] [answered]
I would like to know how to use this when considering managing multiple servers.  Is it one dokploy dashboard per host, or is there some rollup feature?

Thanks

**Comment by Siumauricio** [comment]
Hi, @seigel 

There is currently an open issue #1

Dokploy under the hood uses docker swarm, so it shouldn't be that difficult to implement that functionality, but I'm currently focused on fixing bugs and such, if you have a good idea of ​​how multi-node support should work, it would be very helpful. It helps to comment on the issue and this way it could speed up the development of this functionality.

**Reply by seigel** [reply]
I will see if I can come up with anything.  Thanks!

**Reply by Siumauricio** [reply]
Check here Multi Node support https://github.com/Dokploy/dokploy/pull/96

**Comment by Siumauricio** [comment, marked as answer]
Multi node support is already done you can try out!
